packages/orchestration/src/examples/README.md

```diff
@@ -0,0 +1,17 @@
+# Examples
+
+This directory contains sample contracts showcasing the Orchestration API. Each example demonstrates specific functionalities:
+
+- **basic-flows.contract.js**: Account creation and query sending
```

Comment:
As an FYI, I am going to remove "query sending" as part of #1004. The [git ref](https://github.com/Agoric/agoric-sdk/pull/10004/commits/930732bca37f47dcf48b6702eb141eccb9195942) might change, so here's a copy of the commit message that explains the reasoning:

> refactor: move query invitations to query-flows.contract.js
> - the query invitations that post to vstorage are not representative of an example contract
> - move these invitations to a new test/fixtures/query-flows.contract.js so basic-flows.contact.js
  can stand on its own as an example
  
  
I expect we'll have better examples of querying (or a path for it) after https://github.com/Agoric/agoric-sdk/issues/9796